static/js/lib/search.js

```diff
@@ -192,7 +227,11 @@ export const buildSearchQuery = ({
     builder.sort(field, option)
   }
 
-  const types = type ? [type] : ["comment", "post", "profile"]
+  const types = type
+    ? Array.isArray(type)
+      ? type
+      : [type]
+    : ["comment", "post", "profile"]
```

Comment:
if possible it might be good to refactor away from a nested ternary, just a bit hard to follow

Reply:
✔️